server/services/activitiesService.ts

```diff
@@ -340,4 +345,20 @@ export default class ActivitiesService {
   async createBulkAppointment(appointment: BulkAppointmentsRequest, user: ServiceUser): Promise<BulkAppointment> {
     return this.activitiesApiClient.postCreateBulkAppointment(appointment, user)
   }
+
+  async getDeallocationReasons(user: ServiceUser) {
+    return this.activitiesApiClient.getDeallocationReasons(user)
+  }
+
+  async deallocateFromActivity(deallocateJourney: DeallocateFromActivityJourney, user: ServiceUser) {
+    const request: PrisonerDeallocationRequest = {
+      prisonerNumbers: deallocateJourney.prisoners.map(p => p.prisonerNumber),
+      // eslint-disable-next-line @typescript-eslint/ban-ts-comment
```

Comment:
This is my doing I could do with talk to someone about this (and which date function to use for conversion).